<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lecteurvue</class>
 <widget class="QMainWindow" name="lecteurvue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>lecteurvue</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelDiapoCourant_D">
      <property name="text">
       <string>&lt;nom du diaporama courant&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelDiapoCourant_S">
      <property name="text">
       <string>Diaporama courant :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxImage">
      <property name="title">
       <string>Image &lt;image courante&gt; sur &lt;nombre d'images du diaporama&gt;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="labelIntitule_D">
         <property name="text">
          <string>&lt;intitulé de l'image&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelImageCat_D">
         <property name="text">
          <string>&lt;catégorie de l'image&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>180</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxActions">
      <property name="title">
       <string>Actions</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QPushButton" name="btnReculer">
           <property name="maximumSize">
            <size>
             <width>9999999</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Reculer</string>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="btnAvancer">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Avancer</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="btnLancer">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Lancer le diaporama</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="4">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="3">
          <widget class="QPushButton" name="btnArreter">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Arrêter le diaporama</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>1</width>
             <height>15</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>622</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionFichierOuvrirDiapo"/>
    <addaction name="actionFichierChangerDiapo"/>
    <addaction name="actionFichierEnleverDiapo"/>
    <addaction name="separator"/>
    <addaction name="actionFichierQuitter"/>
   </widget>
   <widget class="QMenu" name="menuParam_tres">
    <property name="title">
     <string>Paramètres</string>
    </property>
    <widget class="QMenu" name="menuVitesse_de_d_filement_2">
     <property name="title">
      <string>Vitesse de défilement</string>
     </property>
     <addaction name="actionParamVitUn"/>
     <addaction name="actionParamVitDeux"/>
     <addaction name="actionParamVitTrois"/>
     <addaction name="actionParamVitPerso"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="menuVitesse_de_d_filement_2"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionAideAPropos"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuParam_tres"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionFichierQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionFichierOuvrirDiapo">
   <property name="text">
    <string>Ouvrir un diaporama</string>
   </property>
  </action>
  <action name="actionFichierChangerDiapo">
   <property name="text">
    <string>Changer de diaporama</string>
   </property>
  </action>
  <action name="actionFichierEnleverDiapo">
   <property name="text">
    <string>Enlever le diaporama</string>
   </property>
  </action>
  <action name="actionAideAPropos">
   <property name="text">
    <string>A propos de</string>
   </property>
  </action>
  <action name="actionParamVitUn">
   <property name="text">
    <string>1 seconde</string>
   </property>
  </action>
  <action name="actionParamVitDeux">
   <property name="text">
    <string>2 secondes</string>
   </property>
  </action>
  <action name="actionParamVitTrois">
   <property name="text">
    <string>3 secondes</string>
   </property>
  </action>
  <action name="actionParamVitPerso">
   <property name="text">
    <string>Vitesse personnalisée</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
